admin/base/settings/defaults.py

```diff
@@ -31,10 +31,16 @@
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.messages',
+    'django.contrib.sessions',
     'django.contrib.staticfiles',
+    'admin.base',
```

Comment:
`admin.base` shouldn't be an app based on the [django docs](https://docs.djangoproject.com/en/1.8/intro/tutorial01/#s-creating-a-project). I imagine it will work as one but I predict bad things in the future if it stays that way.